lib/hacluster.pm

```diff
@@ -123,7 +123,7 @@ sub get_my_ip {
 }
 
 sub get_node_number {
-    return script_output 'crm_mon -1 | awk \'/ nodes configured/ { print $1 }\'';
+    return (is_sle('>=15-sp2')) ? script_output 'crm_mon -1 | awk \'/ nodes configured/ { print $2 }\'' : script_output 'crm_mon -1 | awk \'/ nodes configured/ { print $1 }\''
```

Comment:
Maybe easier to read that way:

    my $index = is_sle('15-sp2+') ? 2 : 1;
    return script_output "crm_mon -1 | awk '/ nodes configured/ { print \$$index }'";

Reply:
Totally agree, thanks.